edx_proctoring/templates/proctored_exam/entrance.html

```diff
@@ -20,15 +20,18 @@
 
     <i class="fa fa-arrow-circle-right"></i>
   </button>
-  <button class="gated-sequence start-timed-exam" data-attempt-proctored=false>
+
+  {% if allow_unproctored %}
```

Comment:
Same comment around `unproctored`.  I'd recommend doing a search and fixing all instances once you have an and appropriate name (e.g. `allow_proctoring_opt_out`).